app/routes/photos/get.js

```diff
@@ -0,0 +1,46 @@
+import config from 'config';
+import Boom from '@hapi/boom';
+import Joi from '@hapi/joi';
+import { getPhoto, photoToJson } from '../../models/photos';
+import logger from '../../services/logger';
+
+const idLength = config.get('idLength');
+
+/**
+ * @apiGroup Photos
+ *
+ * @api {get} /photos/:id  4. GET /photos/:id
+ * @apiDescription Get photo, must be owner or admin.
```

Comment:
Do we need to restrict GET calls for photos to owners / admins? @vgeorge

Reply:
In fact, the only restriction now is being a logged user, so everyone can we all photos. Let's clarify in a separated issue what is the proper data strategy.